src/main/java/com/synopsys/integration/detect/lifecycle/run/step/ThreatIntelScanStepRunner.java

```diff
@@ -23,61 +23,61 @@
 import com.synopsys.integration.rest.response.Response;
 import com.synopsys.integration.util.NameVersion;
 
-public class RlScanStepRunner {
+public class ThreatIntelScanStepRunner {
     
     private final Logger logger = LoggerFactory.getLogger(this.getClass());
     private final NameVersion projectNameVersion;
     private final String projectGroupName;
     private UUID scanId;
     private final OperationRunner operationRunner;
-    private final File rlRunDirectory;
+    private final File threatIntelRunDirectory;
     private final BlackDuckRunData blackDuckRunData;
     private String codeLocationName;
     private static final String STORAGE_UPLOAD_ENDPOINT = "/api/storage/rldata/";
     private static final String STORAGE_RL_CONTENT_TYPE = "application/vnd.blackducksoftware.rl-data-1+octet-stream";
```

Comment:
Can't remember from the discussions, but do we anticipate storage endpoints to change?